CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contributing to Poutyne
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase. We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
```

Comment:
If you've added new code, then it **must** be tested. 
­

Reply:
```suggestion
2. If you've added code that should be tested, you **must** ensure it is properly tested.
```